mtgjson5/providers/tcgplayer.py

```diff
@@ -276,11 +276,33 @@ def generate_mtgjson_sealed_product_objects(
 
         mtgjson_sealed_products = []
 
+        # Load cache if not loaded
+        with constants.RESOURCE_PATH.joinpath("booster_box_size_overrides.json").open(encoding="utf-8") as f:
+            booster_box_size_overrides = json.load(f)
+
+        code = self.code.upper()
+
         for product in sealed_data:
             sealed_product = MtgjsonSealedProductObject()
             sealed_product.name = product["cleanName"]
             sealed_product.identifiers.tcgplayer_product_id = str(product["productId"])
             sealed_product.release_date = product["presaleInfo"].get("releasedOn")
+
+            # XXX: this should be wrapped up in a category field
+            if "booster box" in product["cleanName"].lower():
+                if "set" in product["cleanName"].lower():
+                    sealed_product.booster_box_size = 30
+                    if code in booster_box_size_overrides["set"].keys():
```

Comment:
`.keys()` isn't necessary anymore, as it's default behavior (Mypy should've caught this)